lib/one_model.ex

```diff
@@ -203,7 +203,7 @@ defmodule OneModel do
           #{@schema}.list_by(field1: value1, select: :field2)
           #{@schema}.list_by(field2: value2, select: [:field1, :field3], limit: 2)
       """
-      @spec list_by(keyword()) :: [t()]
+      @spec list_by(keyword()) :: [t()] | [any()]
```

Comment:
Under what circumstance will this return a list of something other than t()? Do we really want it to? I think we expect the model type only. What else can it be?